File: python/1760A - Medium Number.py
n = int(input())
maximum = 0
minimum = 0

for i in range(n):
    a,b,c = map(int,input().split())

    maximum = max(a, max(b, c))
    minimum = min(a, min(b, c))

    if a != maximum and a != minimum:
        print(a)
    elif b != maximum and b != minimum:
        print(b)
    elif c != maximum and c != minimum:
        print(c)

File: C++/1760A - Medium Number.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n, a, b, c, maximum, minimum;

    cin >> n;

    while (n--)
    {
        cin >> a >> b >> c;

        maximum = max(a, max(b, c));
        minimum = min(a, min(b, c));

        if (a != maximum && a != minimum)
        {
            cout << a << endl;
        }
        else if (b != maximum && b != minimum)
        {
            cout << b << endl;
        }
        else if (c != maximum && c != minimum)
        {
            cout << c << endl;
        }
    }

    return 0;
}
